details/ReadonlyREST-api.md

```diff
@@ -0,0 +1,12 @@
+---
+description: readonlyREST API
+---
+
+# readonlyREST API
+
+The readonlyREST OpenAPI documentation provides developers with a comprehensive guide and reference for integrating and interacting with the readonlyREST API. 
+It serves as a detailed resource, offering in-depth explanations of the API endpoints, request parameters, response structures, and supported methods.
+
+{% swagger src=".gitbook/assets/ReadonlyREST-api.yml" path="/api/ror/settings" method="post" %}
```

Comment:
how is it going to be rendered? 
We can use the redoc demo to render it nicely too: 

https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/beshu-tech/readonlyrest-docs/59df014fa136cc8b5c058c710c9e9957dedb03d3/.gitbook/assets/ReadonlyREST-api.yml

Reply:
It will be rendered this way https://docs.gitbook.com/content-creation/blocks/openapi

Reply:
ReadonlyREST capital R

Reply:
> ReadonlyREST capital R

Done ✔️